include/miopen/miopen.h

```diff
@@ -6582,6 +6582,111 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+// FusedAdam APIs
+/** @addtogroup optimizer
```

Comment:
Maybe 
```suggestion
/** @addtogroup SGD
```
The function name (`miopenFusedAdam`) will tell the rest.

Reply:
[resolved]